index.js

```diff
@@ -250,14 +249,14 @@ Notifications.localNotificationSchedule = function(details) {
     }
 
     const iosDetails = {
+      id: (!details.id ? Math.floor(Math.random() * Math.pow(2, 32)).toString() : details.id),
       fireDate: details.date.toISOString(),
-      alertTitle: details.title,
-      alertBody: details.message,
-      category: details.category,
+      title: details.title,
+      body: details.message,
       soundName: soundName,
+      category: details.category,
       userInfo: details.userInfo,
       repeatInterval: details.repeatType,
```

Comment:
I changed it to `repeats`, and updated the documentation to state that 'day' is the only available option for `repeatType` on iOS. This would be another breaking change.

It is possible for the iOS library to specify other repeat intervals, but as [explained here](https://stackoverflow.com/a/47237351/140465) it is not as straightforward. For example, I could not set a fire date of 8 days from now and a weekly repeat; it would instead start firing 1 day from now.